### DISCUSSION TITLE
Zapret выключается на keenetic

### DISCUSSION THREAD

**Discussion by Chivapches** [bol-van/zapret#864] [Q&A]
Обновил zapret до v68, нашел рабочий конфиг ютуб работает инстаграм работает, но через какое-то время просто перестает работать ютуб и инстаграм. Прописываешь /opt/zapret/init.d/sysv/zapret start и все опять начинает работать, как можно определить в чем причина?

**Comment by bol-van** [comment]
нужен хук , иначе прошивка удаляет iptables. Вам в темы дискуссий про кинетик, я не занимаюсь поддержкой кинетиков

**Comment by IndeecFOX** [comment]
@Chivapches 
https://github.com/bol-van/zapret/discussions/801

**Comment by andreysiznet** [comment]
Добрый день подскажите ютуб работает ровно 59 сек! перепробовал все варианты, на одних вообще не работает, на тех которые работают 59 сек

**Comment by andreysiznet** [comment]
система Debian12

**Comment by IndeecFOX** [comment]
> Добрый день подскажите ютуб работает ровно 59 сек! перепробовал все варианты, на одних вообще не работает, на тех которые работают 59 сек

Как-то больше инфы что ли давайте. Вот вы в теме кинетика. У вас на кинетике стоит zapret?
Варианты чего? Стратегий? Как дела на других устройствах в сети?

**Comment by andreysiznet** [comment]
Нет не кенетик написал же Debian12 ,на других устройствах также, ровно минута! при включении если мотать до минуты,воспроизводиться! минута и более висит

**Comment by bol-van** [comment]
Могли отрубить сам гугловский сервер от внешней сети. Тоже что-то резать там дальше

**Comment by andreysiznet** [comment]
> Могли отключить сам сервер Google от внешней сети. Тоже что-то там резали

Интересно что мы берем канал у одного оператора! у одних работает у других(причем как у меня проблема видео 1 мин)!
и через впн тоже !

**Comment by andreysiznet** [comment]
> > Могли отключить сам сервер Google от внешней сети. Тоже что-то там резали
> 
> Интересно, что мы берём канал у одного оператора! У одних работает, у других (причем, как у меня, проблема с видео 1 мин)! и через VPN тоже!

сервисы гугл вообще своей жизнью живут

**Comment by bol-van** [comment]
А вы в курсе, что ТСПУ фильтрует GGC и с внешней стороны тоже ? Ему все равно обращаются к ним из Нидерландов или дядя вася из Ханты-Мансийска.
Они могут сами по себе испытывать проблемы, связанные или не связанные с блокировкой.
На канал после GGC во внешнюю сеть никак юзер повлиять не может.
Но можно попробовать выявить проблемные GGC и забить их в hosts как 127.0.0.1

**Comment by andreysiznet** [comment]
оооо это уже прикольно! подскажите как?
